plugin-server/src/utils/metrics.ts

```diff
@@ -12,13 +12,7 @@ export function instrumentQuery<T>(
     tag: string | undefined,
     runQuery: () => Promise<T>
 ): Promise<T> {
-    const end = dataStoreQueryDuration
```

Comment:
This was unused, plus it wouldn't even work because `instrument` below isn't `await`ed.

I track the metric below now and use a Summary since they're easier to use.